Compile error

on Arch linux (3.14.0-26-ARCH) for armv7l:
```
$ make
make  all-recursive
make[1]: Entering directory '/mnt/teneo'
Making all in src
make[2]: Entering directory '/mnt/teneo/src'
  CC       passphrase_identity-buffer.o
  CC       passphrase_identity-buffer_writer.o
  CC       passphrase_identity-main.o
main.c: In function 'main':
main.c:90:73: error: comparison is always true due to limited range of data type [-Werror=type-limits]
     while ((option = getopt_long(argc, argv, "gshu:p:", options, NULL)) != -1)
                                                                         ^~
cc1: all warnings being treated as errors
make[2]: *** [Makefile:475: passphrase_identity-main.o] Error 1
make[2]: Leaving directory '/mnt/teneo/src'
make[1]: *** [Makefile:399: all-recursive] Error 1
make[1]: Leaving directory '/mnt/teneo'
make: *** [Makefile:331: all] Error 2
$ git status
On branch develop
Your branch is up-to-date with 'origin/develop'.
 
nothing to commit, working tree clean.
$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/armv7l-unknown-linux-gnueabihf/7.2.0/lto-wrapper
Target: armv7l-unknown-linux-gnueabihf
Configured with: /build/gcc/src/gcc/configure --prefix=/usr --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://github.com/archlinuxarm/PKGBUILDs/issues --enable-languages=c,c++,fortran,go,lto,objc,obj-c++ --enable-shared --enable-threads=posix --with-system-zlib --with-isl --enable-__cxa_atexit --disable-libunwind-exceptions --enable-clocale=gnu --disable-libstdcxx-pch --disable-libssp --enable-gnu-unique-object --enable-linker-build-id --enable-lto --enable-plugin --enable-install-libiberty --with-linker-hash-style=gnu --enable-gnu-indirect-function --disable-multilib --disable-werror --enable-checking=release --enable-default-pie --enable-default-ssp --host=armv7l-unknown-linux-gnueabihf --build=armv7l-unknown-linux-gnueabihf --with-arch=armv7-a --with-float=hard --with-fpu=vfpv3-d16
Thread model: posix
gcc version 7.2.0 (GCC)
```

--- a/src/main.c
+++ b/src/main.c
@@ -64,7 +64,7 @@ static struct option options[] = {
 
 int main(int argc, char *argv[])
 {
-    char option;
+    int option;
     bool success = true;
 
     // Output formats.
@@ -78,7 +78,7 @@ int main(int argc, char *argv[])
     size_t username_length = 0;
 
     // Output directory.
-    char *output_directory = NULL;
+    char *output_directory = ".";
 
     // Passphrase.
     char passphrase[1024];

